src/humanio.c

```diff
@@ -184,18 +194,48 @@ int SARHumanLoadFromFile(sar_human_data_struct *hd, const char *filename)
 	    else if(!strcasecmp(buf, "Mass"))
 	    {
 		FGetValuesF(fp, value, 1);
+		
+		/* Mass for a 1.90m tall human was set to 54kg in original
+		 * "Version 1.0" human.ini file. It was certainly an error, so
+		 * let's overwrite it to 90kg.
+		 */
+		if(version_major == 1 && version_minor == 0)
+		    value[0] = 90;
+		
 		if(hdp_ptr != NULL)
 		{
 		    hdp_ptr->mass = (float)MAX(value[0], 0.0);	/* kg */
 		}
 	    }
+	    /* Gender */
+	    else if(!strcasecmp(buf, "Gender"))
+	    {
+		/* Male is default gender.
+		* If Female, a breast will be drawn on torso.
+		*/
+		
+		char *s = FGetString(fp);
+		if( (s != NULL) && (hdp_ptr != NULL) )
+		{
+		    if( !strcasecmp(s, "Female") || !strcasecmp(s, "female") )
+			hdp_ptr->preset_entry_flags |= SAR_HUMAN_FLAG_GENDER_FEMALE;
+		    else
+			/* default */
+			hdp_ptr->preset_entry_flags &= ~SAR_HUMAN_FLAG_GENDER_FEMALE;
+		}
+		free(s);
+	    }
 	    /* AssistingHumans */
 	    else if(!strcasecmp(buf, "AssistingHumans"))
 	    {
 		FGetValuesF(fp, value, 1);
 		if(hdp_ptr != NULL)
 		{
-		    hdp_ptr->assisting_humans = MAX((int)value[0], 0);
+		    hdp_ptr->assisting_humans =
+		    (
+			(MAX((int)value[0], 0) < SAR_ASSISTING_HUMANS_MAX)
+			? MAX((int)value[0], 0) : SAR_ASSISTING_HUMANS_MAX
```

Comment:
There is a `CLIP` function for this, if I'm understanding right. `CLIP((int)value[0], 0, SAR_ASSISTING_HUMANS)`

Reply:
You're right, done!